File: dojo/i.b/FizzBuzz/JavaScript/10-30-2013/fizzBuzz.js
<script>
	function isMultipleOfOrContains(i,j)
	{
		if(i%j == 0 )
		{return true;}
		if(String(i).indexOf(String(j)) !== -1)
		{return true;}
		return false;
	}
	
	function fizzBuzz()
	{
        var isFizz = false;
		var isBuzz = false;
		
        for (var i = 1; i < 101; i++) {
            isFizz= isMultipleOfOrContains(i,3);
			isBuzz = isMultipleOfOrContains(i,5);
			
			if(isFizz && isBuzz)
			{ document.write("FizzBuzz <br/>"); }
			else if(isFizz)
            { document.write("Fizz <br/>"); }
            else if (isBuzz)
            { document.write("Buzz <br/>"); }
            else
            { document.write(i + "<br/>"); }
        }
	}
</script>

File: dojo/i.b/FizzBuzz/C#/10-30-2013/fizzBuzz.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace fizzBuzz.console
{
    class fizzBuzz
    {
        static void Main(string[] args)
        {
            for (var i = 1; i < 101; i++)
            {
                var isFizz = IsFizz(i);
                var isBuzz = IsBuzz(i);

                if (isFizz && isBuzz)
                {
                    Console.WriteLine("FizzBuzz");
                }
                else if(isFizz)
                {
                    Console.WriteLine("Fizz");
                }
                else if (isBuzz)
                {
                    Console.WriteLine("Buzz");
                }
                else
                {
                    Console.WriteLine(i.ToString());
                }
            }
            var x = Console.ReadLine();
        }

        private static bool IsFizz(int value)
        {
            return value % 3 == 0 || value.ToString().Contains("3");
        }

        private static bool IsBuzz(int value)
        {
            return value % 5 == 0 || value.ToString().Contains("5");
        }
    }
}
